packages/paste-core/components/tabs/__tests__/tabs.test.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {axe} from 'jest-axe';
+import {render, screen} from '@testing-library/react';
+import {HorizontalTabsExample, VerticalTabsExample} from '../stories/index.stories';
+
+describe('Tabs', () => {
+  describe('Render', () => {
+    it('horizontally and vertically', () => {
+      const {asFragment} = render(
+        <>
+          <HorizontalTabsExample />
+          <VerticalTabsExample />
+        </>
+      );
+      expect(asFragment()).toMatchSnapshot();
+    });
+
+    it('relevant html and aria attributes', () => {
+      render(<HorizontalTabsExample />);
+
+      const renderedTabList = screen.getByRole('tablist');
+      expect(renderedTabList.getAttribute('aria-orientation')).toEqual('horizontal');
+      expect(renderedTabList.getAttribute('aria-label')).toEqual('My tabs');
+
+      const renderedTabs = screen.queryAllByRole('tab');
+      renderedTabs.forEach(renderedTab => {
+        expect(renderedTab.getAttribute('aria-controls')).toBeDefined();
```

Comment:
Is it worth testing the values of these?

For this component to be accessible we know that `aria-controls` should match the id of the panel that it shows. We know `aria-selected` should only be true when the tab is selected. `tabindex` should only be `0` also when it is selected. We know the panel should be `aria-labelledby` the tab that controls it.

That way if we bump the package and it regresses we the accessibility, we'll know about it